lib/ansible/modules/system/pamd.py

```diff
@@ -186,16 +189,37 @@
 """
 
 RETURN = '''
-dest:
-    description: path to pam.d service that was changed
+change_count:
+    description: How many rules were changed
+    type: int
+    sample: 1
     returned: success
+new_rule:
+    description: The changes to the rule
     type: string
-    sample: "/etc/pam.d/system-auth"
+    sample: None      None None sha512 shadow try_first_pass use_authtok
+    returned: success
+updated_rule_(n):
+    description: The rule(s) that was/were changed
+    type: string
+    sample:
+    - password      sufficient  pam_unix.so sha512 shadow try_first_pass
+      use_authtok
+    returned: success
+result:
```

Comment:
result is too close to 'results' which we use internally, i recommend changing the name